Misc/My Room/sp_greeting.csv

```diff
@@ -173,10 +173,10 @@ HELLO_REPORT_FAV2#7,"""……報告があるんだけど"""
 HELLO_REPORT_FAV1#7,"""HELLO_REPORT_FAV1<br>Debgu_error"""
 HELLO_REPORT_FAV0#7,"""HELLO_REPORT_FAV0<br>Debgu_error"""
 HELLO_OTHER#7,"""……よく来たな"""
-GOODBYE_OTHER#7,"""ふん……それじゃあな"""
+GOODBYE_OTHER#7,"""Well……See ya."""
```

Comment:
Replace Japanese punctuation like this with English equivalents. In this case, `……` for `...`